src/components/Blocking/ItemListLarge.js

```diff
@@ -12,15 +12,15 @@ const ItemListLarge = ({label, desc, iconReght, icon, onPress}) => {
   return (
     <TouchableOpacity style={styles.container} onPress={onPress}>
       <View style={styles.header}>
-        {icon === 'block' && <IconEn name="block" size={17} color={COLORS.black} />}
+        {icon === 'block' && <IconEn name="block" size={17} color={COLORS.white2} />}
         {icon === 'handcuffs' && <Handcuffs />}
         <Text style={styles.label}>{label}</Text>
       </View>
       <View style={styles.content}>
         <Text style={styles.desc}>{desc}</Text>
         {iconReght && (
           <TouchableOpacity style={styles.btn}>
-            <IconFA5 name="chevron-right" size={17} color={COLORS.black} />
+            <IconFA5 name="chevron-right" size={17} color={COLORS.gray500} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are primarily color changes for the icons and text. The `IconEn` component's color has been changed from black to white2, and the `IconFA5` component's color has been changed from black to gray500. These changes seem to be in line with the PR's purpose of changing colors on all screens except feed staging. However, please ensure that these color changes do not affect the visibility and readability of the icons and text, especially considering different screen brightness settings and potential color blindness issues among users.
```diff
- {icon === 'block' && <IconEn name="block" size={17} color={COLORS.black} />}
+ {icon === 'block' && <IconEn name="block" size={17} color={COLORS.white2} />}
- <IconFA5 name="chevron-right" size={17} color={COLORS.black} />
+ <IconFA5 name="chevron-right" size={17} color={COLORS.gray500} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->